amgut/db/patches/0009.sql

```diff
@@ -0,0 +1,229 @@
+--Create pet survey in database
+----------------------------------------------------------
+-- survey_group
+----------------------------------------------------------
+INSERT INTO survey_group (group_order, american, british) VALUES (-2, 'Pet Information', 'Pet Information');
```

Comment:
Arbitrarily chosen. Needs to be one that human surveys doesn't use, and human starts at -1, so I put pets at -2.